SeaPublicWebsite.BusinessLogic/Models/Enums/EpcHeatingType.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace SeaPublicWebsite.BusinessLogic.Models.Enums;
+
+public enum EpcHeatingType
+{
+    GasBoiler,
+    OilBoiler,
+    LpgBoiler,
+    Storage,
+    DirectActionElectric,
+    HeatPump,
+    Other,
+    DoNotKnow,
+    Biomass,
+    CoalOrSolidFuel,
+}
```

Comment:
Instead of storing two heating and other heating type enums, it is much **much** easier to work with just one